src/store/profile/index.ts

```diff
@@ -36,6 +47,136 @@ export const profileSlice = createSlice({
 			state.errorLoading = action.payload;
 			state.currentRequestId = undefined;
 		},
+		[fetchRequisites.fulfilled.type]: (state: IState, action: PayloadAction<IRequisitesResponse>) => {
+			state.loading = false;
+			state.requisites = action.payload.data;
+			state.currentRequestId = undefined;
+		},
+		[fetchRequisites.pending.type]: (state: IState, action: PayloadAction<string, string, { requestId: string }>) => {
+			state.loading = true;
+			state.errorLoading = '';
```

Comment:
И везде поправь со строки на null
```suggestion
			state.errorLoading = null;
```